src/hints/execution.rs

```diff
@@ -1560,8 +1599,8 @@ pub fn write_old_block_to_storage(
     let mut execution_helper: ExecutionHelperWrapper = exec_scopes.get(vars::scopes::EXECUTION_HELPER)?;
 
     let block_hash_contract_address = constants
-        .get(BLOCK_HASH_CONTRACT_ADDRESS)
-        .ok_or_else(|| HintError::MissingConstant(Box::new(BLOCK_HASH_CONTRACT_ADDRESS)))?;
+        .get(vars::constants::BLOCK_HASH_CONTRACT_ADDRESS)
```

Comment:
Is it more idiomatic?. Just asking. For me it just makes reading code more difficult (difficult = more characters to visually parse, longer lines). There are very few name conflicts.

Reply:
Mmh for this one I should use the new `get_constant()` utility function anyway.

Reply:
But yes, I think that specifying the module explicitly here makes the code clearer.